main.js

```diff
@@ -160,4 +160,42 @@ const inventory = [
     originalStock: 10,
     sold: 8,
   },
-];
\ No newline at end of file
+];
+
+// Opdracht 1a
+const tvTypes = inventory.map((stock) => {
+  return stock.type;
+});
+
+console.log(tvTypes);
+
+// Opdracht 1b
+const SoldOutTvs = inventory.filter((stock) => {
+  if(stock.originalStock - stock.sold === 0){
+    return stock;
+  }
+});
+
+console.log(SoldOutTvs);
+
+// Opdracht 1c
+const tvWithAmbilight = inventory.filter((stock) => {
+  if(stock.options.ambiLight === true){
+    return stock;
+  }
```

Comment:
Omdat `stock.options.ambiLight` op zichzelf al een boolean is hoef je deze niet te checken op `true`
```suggestion
  return stock.options.ambiLight;
```